src/test/java/org/openstreetmap/atlas/geography/atlas/delta/AtlasDeltaEdgeTest.java

```diff
@@ -38,7 +38,8 @@ public void testAdded()
         final Atlas alter = alterBuilder.get();
 
         final SortedSet<Diff> diffs = new AtlasDelta(base, alter).generate().getDifferences();
-        logger.debug("testAdded(): {}", Diff.toString(diffs));
+        logger.info("testAdded(): {}", Diff.toString(diffs));
+        logger.info("testAdded(): {}", Diff.toDiffViewFriendlyString(diffs));
```

Comment:
Any reason this became info from debug?

Reply:
I changed it to see output while debugging. Forgot to change it back :)